apps/users/src/app/controllers/users.controller.v2.ts

```diff
@@ -564,4 +567,60 @@ export class UsersControllerV2 {
 
     return this.userServiceV2.unBlockUser(user, targetCastcleId);
   }
+
+  @CastcleBasicAuth()
+  @Post(':userId/recasts')
+  async recastContent(
+    @Auth() authorizer: Authorizer,
+    @Body() { contentId }: RecastDto,
```

Comment:
```suggestion
    @Body() { commentId }: GetContentDto,
```

ใช้ชื่อนี้ดีกว่า จะได้ใข้กับ likeCast ได้ด้วย ถ้า body ไหนมีเพิ่มก็จะได้ extends ไปใช้ต่อได้